app/src/main/java/com/main/mealplanner/dao/IRecipeDAO.kt

```diff
@@ -10,11 +10,7 @@ import retrofit2.http.GET
 interface IRecipeDAO {
 
     @GET("filter.php?i=")
-    fun getAllRecipes() : Call<RecipeList>
-
-
-    //@Query("SELECT * FROM Recipe")
-    //fun getAllRecipes()  : LiveData<List<Recipe>>
```

Comment:
Good idea.

 Committing commented out code is considered bad practice in version control, because version control offers better alternatives:
1) Use a feature branch for experimental features.
2) Delete what is not needed.  You can always compare with history to see what used to be there.
Commented out code is considered clutter in a version control system.